marathon_acme/tests/helpers.py

```diff
@@ -21,3 +26,13 @@ def request(self, method, uri, headers=None, bodyProducer=None):
         return self._requestWithEndpoint(
             key, self._proxyEndpoint, method, parsedURI, headers,
             bodyProducer, parsedURI.originForm)
+
+
+def fake_client(resource):
+    """
+    Build a Treq HTTPClient aroudn a FakeServer to make requests against the
+    given resource.
+    """
+    fake_server = FakeServer(Site(resource))
+    fake_agent = FakeServerAgent(fake_server.endpoint)
+    return HTTPClient(fake_agent)
```

Comment:
I've tried to swap these lines for just `return StubTreq(resource)` but get a long stack trace from Twisted :(

https://gist.github.com/JayH5/f5872fb2bc63001ed055881dea8f0172